pod2daemon/Makefile

```diff
@@ -15,6 +15,16 @@ BUILD_IMAGES    ?=$(FLEXVOL_IMAGE) $(CSI_IMAGE) $(REGISTRAR_IMAGE)
 
 include ../lib.Makefile
 
+# Calculate a timestamp for any build artefacts.
+DATE:=$(shell date -u +'%FT%T%z')
```

Comment:
Just adding the date here isn't going to get it added to the binary since we also need to add a code variable to store that inforamation. That said, we're internally considering removing build date from our artifacts since it makes the binaries not reproducible, so IMO we should just remove the build date from the new LDFLAGS you're adding in this PR.